### ISSUE TITLE
[7.x] Custom Migration Table with Prefix Error

### ISSUE THREAD

**Issue by giflw** [laravel/framework#32876]
- Laravel Version: 7.6.1
- PHP Version: 7.2.9
- Database Driver & Version: sqlsvr / Microsoft SQL Server 2016 (SP2) (KB4052908)

### Description:

Migrations only works on first run whe using custom schema on database.

### Steps To Reproduce:

Change migration table to 'schemaName.tableName'.

See #31152.

**Comment by thisismhp** [comment]
You can select database/schema with connection name like below

```
Schema::connection('foo')->create('bar', function (Blueprint $table) {
.
.
.
}
```

**Comment by giflw** [comment]
@thisismhp @driesvints the problem is with the migration table name. The migration table is in custom schema: myschema.tbl_migrations. When I run the migrations only the first run works. All others run fail because the table is not found on SQL Server. I'm using the fix on #31152 as workaround
